app/controllers/katello/api/v2/content_view_filter_rules_controller.rb

```diff
@@ -106,7 +106,7 @@ def find_filter
     end
 
     def find_rule
-      rule_clazz = ContentViewFilter.rule_class_for(@filter)
+      rule_clazz = ContentViewFilter.rule_class_for(@filter).where(content_view_filter_id: @filter.id)
```

Comment:
This is kind of a nitpick, but I think worth mentioning:

The `rule_clazz` previously returned a class, such as `Katello::ContentViewPackageFilterRule` but now it returns an Active Record Relation instance, so I think it's misnamed.

One way to clean this up is to put `rule_clazz` back the way it was and define `@rule` as:
```
@rule = rule_clazz..where(content_view_filter_id: @filter.id).find(params[:id])
```

It'll result in the same query to the DB:

```
Katello::ContentViewPackageFilterRule Load (0.2ms)  SELECT  "katello_content_view_package_filter_rules".* FROM "katello_content_view_package_filter_rules" WHERE "katello_conte     nt_view_package_filter_rules"."content_view_filter_id" = $1 AND "katello_content_view_package_filter_rules"."id" = $2 LIMIT $3  [["content_view_filter_id", 696247676], ["id", 1033226503], ["LIMIT", 1]]
```